freqtrade/rpc/telegram.py

```diff
@@ -1368,6 +1368,12 @@ def _whitelist(self, update: Update, context: CallbackContext) -> None:
         try:
             whitelist = self._rpc._rpc_whitelist()
 
+            if context.args:
+                if "sorted" in context.args:
+                    whitelist['whitelist'] = sorted(whitelist['whitelist'])
+                if "nobase" in context.args:
+                    whitelist['whitelist'] = [pair.split("/")[0] for pair in whitelist['whitelist']]
```

Comment:
A currency pair has the following naming: `base/quote[:settle]` (settle only for futures).

Keeping the above in mind, think the selected naming is pretty confusing.

it's called nobase - but in reality, it's removing the quote currency (and potentially the settle currency), leaving the base-currency in place, so either "baseonly" or "noquote" would be more fitting (i'd prefer the positive version though, as the only thing left will be the base).

Reply:
I can't believe I've mixed these two up, especially after referring to them in the correct way on discord, perhaps I second guessed myself. I have changed to `baseonly` following your recommendation.